<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_window</class>
 <widget class="QWidget" name="book_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" rowspan="4">
        <widget class="QLabel" name="bookCover_label">
         <property name="text">
          <string>Cover</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="bookIsbnIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="bookPagesIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="bookPages_label_2">
         <property name="text">
          <string>Pages</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLabel" name="bookDescription_label">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="bookYear_label_2">
         <property name="text">
          <string>Year</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="bookTitle_label">
         <property name="text">
          <string>Title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="bookIsbn_label">
         <property name="text">
          <string>ISBN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="bookYearIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="bookGenre_label">
         <property name="text">
          <string>Genre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="bookGenreIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Edit/Delete</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="editBook_label">
         <property name="text">
          <string>Edit book</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="bookTitle_label_2">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="bookTitle_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="bookIsbn_label_2">
         <property name="text">
          <string>ISBN:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="bookIsbn_lineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="bookAuthor_label_2">
         <property name="text">
          <string>Author:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="bookAuthor_comboBox"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="bookGenre_label_2">
         <property name="text">
          <string>Genre:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="bookGenre_comboBox"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="bookCover_label_2">
         <property name="text">
          <string>Cover:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QPushButton" name="bookPreview_button">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="bookDescription_label_2">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QPlainTextEdit" name="bookDescription_plainTextEdit"/>
       </item>
       <item row="15" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="16" column="1">
        <widget class="QPushButton" name="editBook_button">
         <property name="text">
          <string>Edit book</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QPushButton" name="deleteBook_button">
         <property name="text">
          <string>Delete book</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="uploadCover_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="bookCoverPath_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="bookPages_label">
         <property name="text">
          <string>Pages:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="bookPages_lineEdit"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="bookStatus_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QComboBox" name="bookStatus_comboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="bookYear_label">
         <property name="text">
          <string>Year:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="bookYear_comboBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
